inst/templates/ComparisonBarGraphHOR.R

```diff
@@ -19,19 +21,19 @@ p12 <- "STANDARD OB GYN CLINIC"
 p13 <- "FOO MEDICINE"
 p14 <- "BAR BAZ OB GYN CLINIC"
 
-performers = c(p1, p2, p3, p4, p5, p6, p7,
-               p8, p9, p10, p11, p12, p13, p14)
-performance = c(0.20, 0.20, 0.20, 0.20, 0.40, 0.60, 0.70,
-                0.80, 0.85, 0.90, 0.94, 0.97, 0.99, 1.00)
-performance_labels = c("20/100","20/100","20/100","20/100","40/100","60/100","70/100",
-                       "80/100","85/100","90/100","94/100","97/100","99/100","100/100")
-achievable_benchmark_line = 0.50
-
+#performers = c(p1, p2, p3, p4, p5, p6, p7,
+#               p8, p9, p10, p11, p12, p13, p14)
+#performance = c(0.20, 0.20, 0.20, 0.20, 0.40, 0.60, 0.70,
+#                0.80, 0.85, 0.90, 0.94, 0.97, 0.99, 1.00)
 # Below 20 annotation arrow
 #recipient_id = "XDNU WHC OB GYN CLINIC"
 # Above 20 annotation inside bar graph
 recipient_id = "FOO MEDICINE"
 
+###--------------------------------NOT BEING USED------------^
```

Comment:
Remove unused code.  If we need it again, we can dig it up from the git history.